internal/allocation/channel_bind.go

```diff
@@ -4,15 +4,17 @@ import (
 	"fmt"
 	"net"
 	"time"
+
+	"github.com/gortc/turn"
 )
 
-const channelBindTimeout = time.Duration(10) * time.Minute
+var channelBindTimeout = time.Duration(10) * time.Minute
```

Comment:
Can we make this an argument to the server instead?

That way users can configure via https://github.com/pion/turn/blob/master/turn.go#L23 and then we can keep the default `10` minutes still!

Also would be great to have tests for this. I think we are/were leaking at one point, and we can close the ticket if you can assert that timeouts actually work/tear down everything :)

Reply:
Put `channelBindTimeout ` into start configure params is a good idea, I will have a try.